src/components/Rating/Rating.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+
+class Rating extends Component {
+  constructor(props) {
+    super(props);
+    this.num = Math.round(props.children);
+  }
+
+  renderStars = function () {
+    let ratings = [];
+
+    for (let i = 0; i < this.num; i++) {
+      ratings.push('★');
+    }
+
+    while (ratings.length < 5) {
+      ratings.push('☆');
+    }
+    return ratings;
```

Comment:
Maybe, a good solution could be to use `'★'.repeat(this.num) + '☆'.repeat(5 - this.num)`